.circleci/config.yml

```diff
@@ -1,111 +1,82 @@
 version: 2
 jobs:
   unit:
-    docker: &py36
-      - image: python:3.6
+    docker: &containers
+      - image: fishtownjacob/test-container
```

Comment:
is this built by `Dockerfile` in this repo?

Reply:
Yeah. I use this gross and lame shell script to do it:
```bash
#!/bin/bash
set -xeuo pipefail
commit_message="$1"
shift
if [[ -z ${commit_message} ]]; then
    echo 'no commit message'
    exit 1
fi
docker build --tag 'test-container:latest' .
container_id="$(docker run -it test-container:latest hostname | tr -d '[[:space:]]')"
docker commit -a 'Jacob Beck <jake@fishtownanalytics.com>' -m "${commit_message}" ${container_id} test-container
docker tag test-container:latest fishtownjacob/test-container:latest
docker push fishtownjacob/test-container:latest
```